pom.xml

```diff
@@ -42,6 +42,8 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <jenkins.version>1.651.2</jenkins.version>
```

Comment:
Well I don't think you have to, and I think that if it can be solved in a simple way without bumping the core dep then it should be done, so that other bugfixes and features can be done after this and still benefit users of the older cores.